client.go

```diff
@@ -13,13 +12,12 @@ var (
 	// takes place; access is not synchronised.
 	Client Service = BareClient
 	// RoundTripper is used by default in Typhon clients
-	RoundTripper http.RoundTripper = &httpcontrol.Transport{
+	RoundTripper http.RoundTripper = &http.Transport{
 		Proxy:               http.ProxyFromEnvironment,
 		DisableKeepAlives:   false,
 		DisableCompression:  false,
-		MaxIdleConnsPerHost: 10,
-		DialKeepAlive:       10 * time.Minute,
-		MaxTries:            6}
```

Comment:
This is interesting, I remember noticing this when investigating retries and forgot to bring this up. 

Just so I understand this clearly, for `GET` requests did we potentially have multiple things retrying (eg if Linkerd does retries and it gives up)

Reply:
I _believe_ that this only applied to `net.Dial` failures – ie. a network connection to the destination couldn't be established – not application failures which result in a `5xx` response.